### PR TITLE
Update web teleop and something

### PR DESCRIPTION
**Pull request by dwlee**
- web teleop update
- web remocon update as changeing uuid.js
- set default value in rocon_interaction.js

### CODE REVIEW

File: rocon_web_common/www/js/rocon/gui_libary/interface/video_streamer_interface.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    ros: that.ros,
+    name : that.imageStreamTopicName, 
+    messageType : that.imageStreamTopicType,
+  });
+
+  subImageStream.subscribe(function(msg) {
+    that.imageStreamCallback(msg)
+  })
+
+  that.changeVideoStreamTopic = function(msg){
```
Comment (jihoonl) [inline rocon_web_common/www/js/rocon/gui_libary/interface/video_streamer_interface.js]: This is not used function. please comment what it is for.

File: rocon_web_common/www/js/ros/mjpegcanvas.js [hunk 2]
```diff
@@ -0,0 +1,389 @@
+/**
+ * @author Russell Toris - rctoris@wpi.edu
+ */
+
+var MJPEGCANVAS = MJPEGCANVAS || {
```
Comment (jihoonl) [inline rocon_web_common/www/js/ros/mjpegcanvas.js]: Please comment what has been changed from his original. and let's make PR.

**Comment by jihoonl** [comment]
**Restructuring**
-|
 |- rocon : rocon related modules
 |- ros- robotwebtools : from rwt
         |- wpi_rails  : form wpi
 | - thirdparty : from various sources
 | - interface : ...
 | - ui : ...

**video streamer interface**

Two ways to stream images.  `web_video_server` and `rosbridge`. We might want to have two different implementations to support configurations.

Also please comment unused functions why.

**TODO**

Also we need to update `uuid` uses in `concert_common_web_interfaces`

**Comment by dwlee** [comment]
Update regarding comments

File: rocon_web_common/www/js/ros/mjpegcanvas.js (hunk 2 above)
Comment (dwlee) [inline rocon_web_common/www/js/ros/mjpegcanvas.js]: Change the delimeter (`?` --> `&`)  and PR.

File: rocon_web_common/www/js/rocon/gui_libary/interface/video_streamer_interface.js (hunk 1 above)
Comment (dwlee) [inline rocon_web_common/www/js/rocon/gui_libary/interface/video_streamer_interface.js]: update comment

File: rocon_web_common/www/js/rocon/interactions.js [hunk 3]
```diff
@@ -1,3 +1,7 @@
+/*
```
Comment (jihoonl) [inline rocon_web_common/www/js/rocon/interactions.js:1]: Please make a pr to rocon_tools/rocon_interactions.

**Comment by jihoonl** [comment]
Also please comment out unused code. They may confuse for further development.

File: rocon_web_common/www/js/rocon/interactions.js (hunk 3 above)
Comment (dwlee) [inline rocon_web_common/www/js/rocon/interactions.js:1]: Update and PR
https://github.com/robotics-in-concert/rocon_tools/pull/80

**Comment by dwlee** [comment]
Comment out unused code and add more detail reason why they are commented out.

File: rocon_web_common/www/js/interface/resource_chooser_interface.js [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+  - rocon_web_common/www/js/ros/robotwebtools/roslib.js
+*/
+
+ResourceChooserInterface = function(options){
+  var that = this;
+  options = options || {};
+  that.ros = options.ros;
+  that.captureTimeout = options.captureTimeout || 15000;
+  that.availableTeleopTopicName = options.availableTeleopTopicName || 'teleop/available_teleops';
+  that.availableTeleopTopicType = options.availableTeleopTopicType || 'sensor_msgs/CompressedImage';
```
Comment (jihoonl) [inline rocon_web_common/www/js/interface/resource_chooser_interface.js:14]: Please correct the topic type

File: rocon_web_common/www/js/interface/resource_chooser_interface.js [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+  that.captureResourcePairName = options.captureResourcePairName || 'capture_teleop';
```
Comment (jihoonl) [inline rocon_web_common/www/js/interface/resource_chooser_interface.js:15]: this as well.

File: rocon_web_common/www/js/interface/video_streamer_interface.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+VideoStreamerInterface = function(options){
+  var that = this;
+  options = options || {};
+  
+  // Not yet used
+  // It will be used when implemetation of streaming via rosbridge is finished.
+  /*
+  that.ros = options.ros;
+  that.imageStreamTopicName = options.imageStreamTopicName || 'compressed_image';
```
Comment (jihoonl) [inline rocon_web_common/www/js/interface/video_streamer_interface.js:16]: imageStreamTopic name has been used.

Comment (jihoonl) [inline rocon_web_common/www/js/interface/video_streamer_interface.js:16]: All `compressed_image` should be `image`

**Comment by jihoonl** [comment]
Keyboard layout please.

**Pull request merged**